config/core/configmaps/defaults.yaml

```
@@ -122,9 +122,10 @@ data:
 
     # enable-service-links specifies the default value used for the
     # enableServiceLinks field of the PodSpec, when it is omitted by the user.
+    # This is a tri-state flag with possible values of (true|false|default).
     # See: https://kubernetes.io/docs/concepts/services-networking/connect-applications-service/#accessing-the-service
```

total total nit, but could we swap these lines? As it is it reads to me like the "See: [link]" is referring to the tri-state flag instead of giving more details about `enableServiceLinks`.

```suggestion
    # enableServiceLinks field of the PodSpec, when it is omitted by the user.
    # See: https://kubernetes.io/docs/concepts/services-networking/connect-applications-service/#accessing-the-service
    # This is a tri-state flag with possible values of (true|false|default).
```